### PR TITLE
Org.audmes.audmes

### PR DESCRIPTION
**Pull request by swwa**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by swwa** [comment]
bot, build org.audmes.audmes

File: org.audmes.audmes.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
+{
+    "app-id": "org.audmes.audmes",
```
Comment (hfiguiere) [inline org.audmes.audmes.json:2]: `audmes.org` doesn't exist. This id doesn't meet the requirements.

File: org.audmes.audmes.appdata.xml
Comment (hfiguiere) [inline org.audmes.audmes.appdata.xml:1]: when the app-id is sorted out, this file should be upstreamed.

**Comment by swwa** [comment]
@hfiguiere https://github.com/flathub/flathub/pull/4165 replaces this pull request. Closing.

**Pull request closed without merging**